### PR TITLE
Fix a bug using SuiteSparse on Ubuntu 16.04.

### PR DESCRIPTION
**Pull request by wohe**
Using the catkin "suitesparse" and having libsuitesparse-dev installed
leads to builds which crash. This changes the configuration to suggest
using libsuitesparse-dev.

### CODE REVIEW

**Comment by wohe** [comment]
This is to fix #14.

File: README.md [hunk 1]
```diff
@@ -35,11 +34,10 @@ These dependencies always have to be installed:
     # Clone the necessary repos into your Catkin workspace
     git clone https://github.com/googlecartographer/cartographer.git
     git clone https://github.com/googlecartographer/cartographer_ros.git
-    git clone https://github.com/ethz-asl/ceres_catkin.git
-    git clone https://github.com/ethz-asl/suitesparse.git
-    git clone https://github.com/ethz-asl/glog_catkin.git
-    git clone https://github.com/ethz-asl/gflags_catkin.git
-    git clone https://github.com/ethz-asl/catkin_simple.git
+    git clone https://github.com/ethz-asl/ceres_catkin.git  # Caution! Make sure not to have "suitesparse" in you Catkin workspace
```
Comment (damonkohler) [inline README.md]: # Caution! Make sure you do not have "suitesparse" in you Catkin workspace

**Comment by damonkohler** [comment]
LGTM

**Comment by rohbotics** [comment]
+1

**Pull request merged**